### PR TITLE
Archaeology and France-Stanford Subthemes

### PR DESCRIPTION
**Pull request by josephgknox**
# READY FOR REVIEW

# Summary
- Adds a subtheme for Archaeology and France-Stanford
- Moves above site's specific code out of the suhumsci base theme
- Cleans up lingering site-specific settings and config in base theme

# Needed By (Date)
- End of sprint (10/25)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Sync
3. Go to `admin/appearance` and enable related subtheme; make sure site-specific styles appear

# Affected Projects or Products
- Archaeology
- France-Stanford
- suhumsci (theme)

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/HSD8-461
- https://stanfordits.atlassian.net/browse/HSD8-462

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: docroot/themes/humsci/su_humsci_theme/theme-settings.php [hunk 1]
```diff
@@ -5,29 +5,8 @@
  * Provides an additional config form for theme settings.
  */
 
-use Drupal\Core\Form\FormStateInterface;
-
 /**
  * Implements hook_form_system_theme_settings_alter().
  *
  * Form override for theme settings.
  */
```
Comment (pookmish) [inline docroot/themes/humsci/su_humsci_theme/theme-settings.php]: Thanks for deleteing the function. but also delete the function comment too.

Comment (josephgknox) [reply docroot/themes/humsci/su_humsci_theme/theme-settings.php]: Doh.

Comment (josephgknox) [reply docroot/themes/humsci/su_humsci_theme/theme-settings.php]: Removed.

**Comment by pookmish** [comment]
Lets not forget to change the theme after deployment

**Pull request merged**